sources/packages/web/src/composables/useFormioDropdownLoader.ts

```diff
@@ -189,6 +189,14 @@ export function useFormioDropdownLoader() {
     );
   };
 
+  const loadReportTypes = async (
```

Comment:
What is the benefit of having the method `loadReportTypes`? It looks like a rename from `loadDropdown`?